Gemfile.lock

```diff
@@ -31,6 +30,8 @@ GEM
       diff-lcs (>= 1.1.3, < 2.0)
     rspec-mocks (2.14.4)
     unf (0.1.4)
+      unf_ext
```

Comment:
I *think* this line is what's causing Travis to fail on JRuby. We've had this issue before. See https://github.com/constantcontact/ruby-sdk/commit/408c0661a164c650ccc3edef3439357587c67f16#diff-e79a60dc6b85309ae70a6ea8261eaf95